.claude/guides/harness-guideline.md

```
@@ -0,0 +1,1257 @@
+> 아예 다른 맥락의 작업을 하게 되면 /clear. 이어가야 하면 handoff로 맥락을 넘긴다.
+
+### 7.3 전략적 압축 (Strategic Compaction)
+
+**언제 압축하는가:**
+- 리서치 완료 **후**, 구현 시작 **전** (발견한 것을 정리)
+- 마일스톤 달성 **후**, 다음 페이즈 **전** (상태 정리)
+
+**언제 압축하면 안 되는가:**
+- 구현 **중간** — 작업 중인 변수 스코프, 의사결정 맥락이 유실됨
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)